_pages/clients/dispute.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Dispute
+permalink: /dispute/
+web: https://ethicnology.github.io/dispute/#/
+github: ethicnology/dispute
+instances:
+- name: Dispute on GitHub Pages 
+  url: https://ethicnology.github.io/dispute/#/
+  comment: 
+tags:
+- social
+- client
+- debate
+platforms:
+- desktop
+- mobile
```

Comment:
name the supported OSs. Try to keep the word count low in platforms and tags such that people can filter picking from few options. For example: Apple and iPhone would be bad. Either, or. "mobile" is not helpful as it will not show up when searching for Android.

Reply:
fixed.
This supports: 
- linux
- macos
- windows
- android
- ios
- web